bin/MadGraph5_aMCatNLO/cards/production/13TeV/HToAATo4B/SUSY_WH_WToAll_HToAATo4B_M-MASS/SUSY_WH_WToAll_HToAATo4B_M-MASS_proc_card.dat

```diff
@@ -0,0 +1,6 @@
+import model NMSSM --modelname
+
+generate p p  > w+ h01 , (h01 > a01 a01, a01 > b b~, a01 > b b~)
+generate p p  > w- h01 , (h01 > a01 a01, a01 > b b~, a01 > b b~)
```

Comment:
Here, it should be "add process".
Can you double check for all the other requests you made that this didn't happen? I remember I found one for you but did not look closely enough to the other ones

Reply:
Hi @sihyunjeon ,
Good catch again!  I've fixed it.  WH and ggH+0/1 jet are the only cards with two processes.